portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.js

```diff
@@ -41,6 +59,24 @@ const getFormattedReportPeriod = () => {
   return format(lastMonthDate, 'MMMM yyyy');
 };
 
+/**
+ * Checks if a utilisation report has already been submitted by the specified bank for the current reporting period
+ * @param bank {object} - the bank to check
+ * @returns {Promise<boolean>}
+ */
+const getIsReportSubmitted = async (bank) => {
+  // TODO FN-1164 - check how this endpoint has been updated
```

Comment:
Still need to check this.

Reply:
@ChristianMcc - I think after your recent changes I can still use this function to find all existing reports (I think this was previously point at a `previous-reports` endpoint but now points to the new `/v1/bank/${bankId}/utilisation-reports` one). Let me know if this isn't right.

Reply:
yep this is right, our method of abstraction to the api file actually working out nicely here for once! However you can now query my month and year so it's probably worth doing that query and then checking that exists rather than then checking the response has something that matches the right year/month combo at the end of this function